build.gradle.kts

```diff
@@ -76,6 +76,7 @@ val tools = compiler.get().metadata.installationPath.file("lib/tools.jar")
 dependencies {
     compileOnly(files(tools))
     compileOnly("org.jetbrains:annotations:24.0.+")
+    implementation("org.apache.commons:commons-text:latest.release")
```

Comment:
Hmm; as this is an annotation processor we had intentionally kept dependencies to an absolute minimum; while I understand the appeal of a library; would it be possible to sanitize without this lib?

Reply:
Probably, but I'd rather just vendor the dependency with the shadowjar plugin

Reply:
Resolved